data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -3,14 +3,19 @@ set -euoE pipefail ## -E option will cause functions to inherit trap
 
 . /usr/local/bin/release-image.sh
 
-mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
+mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests,container-logs}
+mkdir --parents /etc/container-logs
```

Comment:
I intend to move this to /car/log/container-logs or some such. Suggestions welcome.